x-pack/filebeat/input/gcppubsub/pubsub_test.go

```diff
@@ -338,10 +339,11 @@ func TestTopicDoesNotExist(t *testing.T) {
 	cfg := defaultTestConfig()
 
 	runTest(t, cfg, func(client *pubsub.Client, input *pubsubInput, out *stubOutleter, t *testing.T) {
-		err := input.run()
-		if assert.Error(t, err) {
-			assert.Contains(t, err.Error(), "failed to subscribe to pub/sub topic")
-		}
+		require.Error(t, input.run())
+		// err := input.run()
+		// if assert.Error(t, err) {
+		// 	assert.Contains(t, err.Error(), "failed to subscribe to pub/sub topic")
+		// }
```

Comment:
```suggestion
```

Reply:
Removed comments. Sorry about the comments. I made code changes pretty long back and forgot to remove them.